R/archivePackages.R

```diff
@@ -19,40 +19,40 @@
 ##'   archivePackages(pkg = "drat")  # archive older copies of just one package
 ##' }
 ##' @author Thomas J. Leeper
-archivePackages <- function(repopath=getOption("dratRepo", "~/git/drat"),
-                            type="source", 
+archivePackages <- function(repodir = getOption("dratRepo", "~/git/drat"),
+                            type = "source", 
                             pkg) {
```

Comment:
In general, it is a bad idea to change existing variable names.

`drat` has been out there for a few years, it is highly possibly that _some_ users refer to this variable by its existing name.  By making this change you are forcing them to change their scripts -- which is why such a change is generally seens as not being worth it.  

Please consider reversing the `repopath` -> `repodir` change.

Reply:
Good point, I agree that the argument in the subfunctions should not be renamed in order to not force people to change their scripts.

I did reset my fork
```
git reset --hard HEAD^
git push origin --force
```
Now the first arguments in `pruneRepo` and `archivePackages` are named `repopath` as they were before.